File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for x in commands:
        i = x[0]
        j = x[1]
        k = x[2]
        
        arr = array[i-1:j]
        arr = sorted(arr)
        
        answer.append(arr[k-1])
    
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for (let i = 0; i < commands.length; i++) {
        let [start, end, idx] = commands[i];
        let sorted =  array.slice(start - 1, end).sort((a, b) => a - b);
        answer.push(sorted[idx - 1]);
    }
    return answer;
}